extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -216,33 +218,33 @@ export class ResourceTypePickerDialog extends DialogBase {
 				if (this.toolRefreshTimestamp !== currentRefreshTimestamp) {
 					return;
 				}
-				let installationRequired = false;
+				let installationNeeded = false;
 				let minVersionCheckFailed = false;
 				const messages: string[] = [];
-				this._toolsTable.data = toolRequirements.map(toolReq => {
-					const tool = this.toolsService.getToolByName(toolReq.name)!;
+				this._toolsTable.data = toolRequirements.map(toolRequirement => {
+					const tool = this.toolsService.getToolByName(toolRequirement.name)!;
 					// subscribe to onUpdateData event of the tool.
 					this._toDispose.push(tool.onDidUpdateData((t: ITool) => {
 						this.updateToolsDisplayTableData(t);
 					}));
 					if (tool.isNotInstalled && !tool.autoInstallSupported) {
-						messages.push(localize('deploymentDialog.ToolInformation', "'{0}' [ {1} ]", tool.displayName, tool.homePage));
```

Comment:
'{0}' is either not installed or not running, You can find more information about the tool at: {1}

Reply:
Fixed.